src/views/Customer/Support.js

```diff
@@ -0,0 +1 @@
+import React from 'react';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)